### PR TITLE
Editorial pass for examples

### PR DESCRIPTION
**Pull request by bumblefudge**
Forgive me if I overstepped my remit (or my understanding) but I tried updating the examples in a couple different ways:
- Made the situations more realistic/concrete and updated `pattern` and `purpose` to match (hopefully in line with the intent of #169 )
- Added a few `limit_disclosure` properties per the updated situations and `purpose` props
- Fixed some date formatting and other minor editorial things while i was there
- Updated the LD example to have a RL2020 status object rather than the somewhat baffling boolean
- Tried to fix a link to the LDP spec on line 2158 (formerly line 2168)

### CODE REVIEW

**Review by brentzundel — APPROVED** [review]

**Comment by bumblefudge** [comment]
fixed the merge conflict (Daniel removed the localization section while I was updating its example like a goof)

**Review by brentzundel — DISMISSED** [review]

**Comment by rado0x54** [comment]
Uh, I should have definitely merged this PR in before doing my own :)

**Comment by rado0x54** [comment]
I would maybe propose to first merge #194. Because it contains the tests that can easily check for the example changes in this one. The other way around may be a little harder, but I can do that as well after the meeting.

**Comment by bumblefudge** [comment]
I'm still a little confused where /test/presentation-submission/example.json is... which leaves me with the unsettling feeling I've made a clerical mistake somewhere 😬

File: spec/spec.md [hunk 1]
```diff
@@ -190,59 +199,7 @@ requirement.
...
-              }
-            }
-          ]
-        }
-
-      }
-    ]
-  }
-}
+[[import ../test/presentation-definition/example.json ]]
```
Comment (csuwildcat) [inline spec/spec.md:202]: I ended up going with `insert`, and the path is assumed to be from the root of the project, so could you change this to : [[insert: ./test/presentation-definition/example.json]]?

File: spec/spec.md [hunk 2]
```diff
@@ -251,72 +208,7 @@ requirement.
...
-                "maximum": "1999-5-16"
-              }
-            }
-          ]
-        }
-      }
-    ]
-  }
-}
+[[import: ../test/presentation-definition/single_group_example.json]]
```
Comment (csuwildcat) [inline spec/spec.md:211]: Same comment above, the form of this is going to be: [[insert: ./test/presentation-definition/single_group_example.json]]

**Review by csuwildcat — CHANGES_REQUESTED** [review]
Just need to get the final format tweak in for the replacer functionality, which looks like this:  `[[insert: ./test/presentation-definition/example.json]]`

**Comment by bumblefudge** [comment]
@csuwildcat does [this](https://github.com/decentralized-identity/presentation-exchange/pull/187/commits/9990430d3c68304ac7bcd8d1fa17bcb0ca88be7b) scratch the itch you've requested?

**Review by csuwildcat — APPROVED** [review]

**Pull request merged**